src/libraries/pkg/Microsoft.Private.PackageBaseline/packageIndex.json

```diff
@@ -83,7 +83,6 @@
         "4.5.0",
         "4.6.0"
       ],
-      "BaselineVersion": "5.0.0",
```

Comment:
Should this say, and be set to e.g. 4.6.0 ? I see other dead-end packages like System.Runtime still have it set.

Reply:
I think its just a matter of cleanup. @joperezr will be better able to answer why we can now delete the baseline version ?

Reply:
What baseline version does is that if any new package is building and has a reference to this assembly, we will force the reference to it to be to the package listed on the BaselineVersion. Keeping it as 5.0.0 would be wrong, as that would mean that we would be elevating references to Microsoft.CSharp to use version 5.0.0 knowing that we will never ship this package. If we remove it, the version of the dependency will be calculated based on latest stable package shipped, and the assembly to package version mapping which are both also here in the index, which will mean that it would already be 4.6.0 so no need to set BaselineVersion. That said, setting it wouldn't hurt either if we wanted to be consistent.